content/tracing/faq/agent-5-tracing-setup.md

```diff
@@ -0,0 +1,70 @@
+---
+title: APM (Tracing) with agent v5
```

Comment:
Why doesn't this page follow the same outline as the other one?

Reply:
👌